### ISSUE TITLE
question

### ISSUE THREAD

**Issue by YanXiao-bit** [yonghanyu/DeepSearch#12]
Dear authors, I am currently using the deepsearch model for training, and the training set is built by myself, using the MS/MS spectra of peptides from modification-enrichment in MassIVE KB v2: MassIVE Knowledge Base spectral library v2 - subset releases experiments, which contain about 110,000 peptides and are about 2G in size, have an accuracy rate of about 85% during training, but the accuracy rate of each round of validation set is only about 54%, is this normal? If I don't achieve the expected effect, is it because my training set is too small and I don't have a good generalization ability?

**Comment by yonghanyu** [comment]
Hi, sorry for the late reply. Such behaviour is not normal, and the size of the dataset can definitely be one of the leading factors of compromised accuracy. I'd fine-tune the provided parameter on your own dataset instead of starting from scratch.

**Comment by YanXiao-bit** [comment]
Hello, dear author, what is the accuracy of the validation set used for the test of the model at the end, thank you for your reply!
